packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -68,15 +160,15 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection impleme
      * @param url - websocket URL
      * @param telemetryLogger - optional telemetry logger
      */
-    public static async create(
+    public static async create2(
```

Comment:
It unfortunately doesn't look like there's a straightforward way to overload this method with different signatures while maintaining the same name without hitting "Class static side 'typeof OdspDocumentDeltaConnection' incorrectly extends base class static side 'typeof DocumentDeltaConnection'."

TS looks like they've had this issue for a bit: https://github.com/microsoft/TypeScript/issues/4628
And the proposed fixes for it have all been closed except for this unmerged one: https://github.com/microsoft/TypeScript/pull/39699

So it looks like either having two separate create functions or matching their signatures

Reply:
There was also this thread on having multiple overloaded definitions defined ahead of time but I'm not sure how that would apply to extending across static classes
https://stackoverflow.com/questions/12688275/is-there-a-way-to-do-method-overloading-in-typescript/12689054#12689054

Reply:
Latest changes made signatures same again, so this is no longer an issue.